### ISSUE TITLE
Problems with binary executable file 

### ISSUE THREAD

**Issue by seriiix** [SharifAIChallenge/AIC21-Client-Python#18]
Using `python -m PyInstaller --onefile Controller.py` with `torch` package will produce a 1GB executable file, And it will take forever to run!

To reproduce the problem try adding `import torch` to  `AI.py` file.

**Comment by MohsenDehghankar** [comment]
Try using `--run-manually` locally to test your client, instead of creating binary.
https://github.com/SharifAIChallenge/AIC21-Server#run-server

**Comment by seriiix** [comment]
Thank you for your response 🙏🏻.

Well, this is not possible because we are running a batch of games automatically we can't manually run our agent when the server requests a new client, But I don't think this is the main problem.

The problem is again the fact that we have to create a binary for a python client. Apart from the fact that the binary is more than 1GB in size, I don't think that your servers can handle 8+ mins of compiling and 6+ mins of starting time for a single client. By saying starting time I mean the amount of the time between starting the binary and binary printing the first `Trying to connect #1` log. Every time a `SARBAZ` or `KARGAR` respawns, the server has to wait for 6 mins straight to run a new instance of the binary file. We ran these tests on a good 8th gen Core i7 processor. So I don't think that creating a binary file for a script-based language like python is necessary as it will not increase the performance. 
**We didn't have these issues in the AIC19 and AIC20** and we hope you fix this problem by changing the server to run the python client from the shell and not the binary.

BTW, We handled the first problem with a hacky solution: We created a binary to run our client from shell (`python Controller.py`) and passed it to the server.
